css-responsive/css-responsive_playground/css/styles.css

```diff
@@ -0,0 +1,121 @@
+/* ==[ PLAYGROUND Challenge! ]== */
+
+/* This challenge includes several smaller challenges and allows you to
+play around with the concepts you've learned. Read the instructions 
+carefully. */
+
+:root {
+  /* Color Variables */
+
+  --color-nemo: #ff4a11;
+  --color-granite: #252629;
+  --color-water-10: #f3f5f9;
+  --color-foam: #ffffff;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: system-ui;
+
+  background-color: var(--color-foam);
+}
+
+h1 {
+  font-size: 3rem;
+  width: 80%;
+  margin: 20px auto;
+  color: var(--color-nemo);
+  text-shadow: 4px 4px var(--color-granite);
+}
+
+.playground {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: auto;
+  padding: 20px;
+  gap: 30px;
+
+  width: 80%;
+  min-height: 100vh;
+}
+
+.section {
+  width: 100%;
+  background-color: var(--color-water-10);
+  padding: 20px;
+}
+
+.box {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: auto;
+}
+
+.box--one {
```

Comment:
Maybe, you've overlooked the task for this selector: 

*"I have a fixed width. Can you replace my fixed width with a relative unit that changes according to the size of my parent element?"*